### PR TITLE
Add changelog

### LINKED ISSUE #99: Start a changelog
For more transparency what's in a release and what not. Ideally this is updated automatically when doing a release.

### PR DESCRIPTION
**Pull request by daniel-mohemian**
Closes #99.

For transparency. Tell me what you think. In the future, we might want to add a Danger check that ensures people add their contributions to the _unreleased_ section and then automatically move them to a release section. Currently, nothing is automated, but I'd say that's still better than no changelog at all.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+[PSPDFKit's Changelog Format](https://pspdfkit.com/blog/2018/the-challenges-of-changelogs/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Start each entry with one of the following keywords: 
+
+- *API*: For highlighting breaking API changes that require people to change their implementations
+- *Adds*: For listing new features
+- *Fixes*: For listing bugfixes
+- *Changes*: For listing improvements other changes
```
Comment (carol-mohemian) [inline CHANGELOG.md]: The description doesn't sound right. 🤔

Comment (markus-mohemian) [inline CHANGELOG.md]: Sounds a bit weird. Maybe add an "and" between improvements and other changes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Include references to issue- or pull-request numbers.
+Use active language and present tense.
+For convenience, feel free to track changes in the *Unreleased* section upon
+merge to master and move them to a respective version upon release.
+
+## Unreleased
+
+- Security: Change fastlane version to fix [#129](https://github.com/airsidemobile/JOSESwift/issues/129) ([#130](https://github.com/airsidemobile/JOSESwift/pull/130))
+- Changes travis to use builting homebrew addon ([#133](https://github.com/airsidemobile/JOSESwift/pull/133))
```
Comment (markus-mohemian) [inline CHANGELOG.md]: "builting"? 🤔 Maybe "builtin"

Comment (daniel-mohemian) [reply CHANGELOG.md]: Good find! 🎖

**Comment by daniel-mohemian** [comment]
Thanks for the review guys! I fixed the wording and the typo. Sorry about that. 😅

**Pull request merged**